### PR TITLE
VTCCA-5185 Registration Complete Contact Details field validation cov…

### PR DESCRIPTION
**Pull request by zachhall1234**
…erage

### CODE REVIEW

File: test/controllers/RegistrationControllerSpec.scala [hunk 1]
```diff
@@ -264,7 +264,36 @@ class RegistrationControllerSpec extends VoaPropertyLinkingSpec with MockitoSuga
 
   }
 
-  "Submitting an invalid individual form" should "return a bad request response" in {
+  "Submitting an individual form with invalid email, mobilePhone, phone, nino" should "return a bad request response" in {
+    val data = Map(
```
Comment (zahiriqbal) [inline test/controllers/RegistrationControllerSpec.scala:268]: You can move this to the FakeObjects class...

Comment (zahiriqbal) [inline test/controllers/RegistrationControllerSpec.scala:268]: You can move this to the FakeObjects class...

File: test/controllers/RegistrationControllerSpec.scala [hunk 2]
```diff
@@ -264,7 +264,36 @@ class RegistrationControllerSpec extends VoaPropertyLinkingSpec with MockitoSuga
...
+    status(res) shouldBe BAD_REQUEST
+    val html = HtmlPage(res)
+    html.shouldContainText("Enter a valid email address")
+    html.shouldContainText("Enter a valid National Insurance number")
+    html.shouldContainText("Telephone number must be between 11 and 20 characters")
+    html.shouldContainText("Enter a telephone number, like 01623 960 001 or +44 0808 157 0192")
+  }
+
+  "Submitting an individual form with non-matching emails" should "return a bad request response" in {
+    val data = Map("email" -> Seq("x@x.com"), "confirmedEmail" -> Seq("1@1.com"))
```
Comment (zahiriqbal) [inline test/controllers/RegistrationControllerSpec.scala:286]: also this

File: test/controllers/RegistrationControllerSpec.scala (hunk 1 above)
Comment (DanSmith98) [reply test/controllers/RegistrationControllerSpec.scala:268]: I don't think this is likely to be used anywhere else, so I'm not sure putting in FakeObjects would be of benefit here.

File: test/controllers/RegistrationControllerSpec.scala (hunk 2 above)
Comment (DanSmith98) [reply test/controllers/RegistrationControllerSpec.scala:286]: I don't think this will be used anywhere else either

**Comment by DanSmith98** [comment]
LGTM :+1:

File: test/controllers/RegistrationControllerSpec.scala (hunk 1 above)
Comment (PeterPerhac) [reply test/controllers/RegistrationControllerSpec.scala:268]: thanks @DanSmith98 you're right, these maps are useful only in this specific place. They belong in the test, not in the FakeObjects

**Comment by PeterPerhac** [comment]
👍  looks good to me

**Pull request merged**